### ISSUE TITLE
[New][LFH] The Virtue DAO: No proposals, Virtuous Swarms On A Mission with Continuous Endowment Funding 

### ISSUE THREAD

**Issue by aquabu** [ethberlinzwei/Find-A-Team#36]
## About Me

Hi, I'm Noah and I've been hacking on responsive organizations, blockchain projects and DAOs since 2014 and startups since 1998. I co-founded a holocratic company that became a [constitutional org governed by pull requests](https://github.com/CoMakery/self-organization-constitution). I currently run [CoMakery.com](https://www.comakery.com) and have done lots of software development on financial, blockchain and web apps. [Github](https://github.com/aquabu) | [LinkedIn](https://www.linkedin.com/in/noahthorp/) | [Twitter](https://twitter.com/noahthorp)

Let me know if you would like to hack with me on a new kind of DAO. One with no proposals, virtuous swarms with a mission, funded by grants and distributing returns on deposits (e.g. [Compound](https://compound.finance/)).

## Why Virtuous Swarms With No Proposals

DAOs help us fulfill a common purpose, allocate resources and coordinate activities.

**But** proposals take a long time to write, read and vote on. Voter turnout is often very low. Waiting for permission for proposals is not fast enough in fast moving scenarios or exploding complexity with lots of localized expertise. Proposals are not in the spirit of permissionless action and do-ocracy. 

## How It Could Work 

As a metaphor, consider a monastic order where individuals gain access to the resources of the order based on their adherence to following the precepts of their order - through their virtue. If the people of the order (monks, knights, artists, activists) remain true to the precepts others who are inspired by their actions and way of acting donate to the endowment.

Here's the rough starting point for the implementation:

* Resource allocation without proposals 
* Funds allocated and made available proportionate to how much a person embodies the virtues that the DAO values
* Funded with DAO grants and compound interest from loaning out deposits using compound (like a coop bank or endowment)
* Paid with a continuous stipend and access for the virtuous with a sustainable distribution of funds - to maintain continuity of virtue incentives
* Blockchain anchored reputation verifying who has virtues that support the values and mission of the DAO
* Monthly allocation of resources with the option to pool resources
* A fair method of sourcing virtue from DAO members or a third party like Kleros
* We will probably build on existing DAO codebase like *DAOStack*

## Who We Need On The Team

* UI/UX Designers - design matters for communicating this idea.
* Developers - Solidity, Javascript.
* Advisors who are interested in the approach or have knowledge about foundations, endowments, and community organization. Pleae join us to help develop the use cases.

## Communication

* Tell me you are interested by responding to this Github task
* Message me on Telegram @noahthorp
* We will setup team chat once the team starts to form (probably telegram)

**Comment by Patrick-T-Anderson** [comment]
Hi @aquabu 
Very interested in applying this approach to physical production in the material world by giving
monks, knights, artists, activists access to land and tools to create the food and shelter they need.

**Comment by aquabu** [comment]
@patrick-t-Anderson absolutely! Their “Virtue” could also unlock access to resources and not just a flow of funds.

**Comment by aquabu** [comment]
@Patrick-T-Anderson are you at the hackathon? Post here or PM me on Telegram @noahthorp

**Comment by Patrick-T-Anderson** [comment]
Hi @aquabu,

I'm not at the hackathon :(

My proposal is at https://github.com/ethberlinzwei/Find-A-Team/issues/23

I will message you on Telegram ;)

**Comment by aquabu** [comment]
Code for this posted here: https://github.com/aquabu/virtue-dao
